app/assets/stylesheets/bootstrap-overrides.scss

```diff
@@ -0,0 +1,31 @@
+/* Bootstrap overrides go here. This is included after bootstrap. */
+
+/* 979px - 980px is where nav-collapse gets triggered */
+@media (max-width: 979px) {
```

Comment:
Good idea, I'll separate out all the responsive stylesheets and make them feature flaggable